apps/verify/views/confirm-email.html

```diff
@@ -13,7 +13,7 @@
           type="text"
           name="confirm-email"
           id="confirm-email"
-          class="form-control{{#className}} {{className}}{{/className}}{{#error}} invalid-input{{/error}}"
+          class="form-control form-control-3-4 {{#className}} {{className}}{{/className}}{{#error}} invalid-input{{/error}}"
```

Comment:
Cool Looks alright, @sulthan-ahmed  there is no other fancy way of doing this in HOF is there?

Reply:
No this is correct.  This is a custom page so @cmedlandho was right to enter this here for this specific scenario